Assets/Scripts/Fighting/AttackCombo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Fighting
+{
+    [CreateAssetMenu(fileName = "Combo", menuName = "ScriptableObjects/AttackCombo", order = 1)]
```

Comment:
```suggestion
    [CreateAssetMenu(fileName = "Combo", menuName = "Fighting/AttackCombo", order = 1)]
```

We shouldn't use a directory called "ScriptableObjects" - we are going to have lots of them. Better sort them by System instantly

Reply:
yes